src/librustc_typeck/check/dropck.rs

```diff
@@ -212,20 +216,40 @@ fn ensure_drop_predicates_are_implied_by_item_defn<'tcx>(
         // the analysis together via the fulfill , rather than the
         // repeated `contains` calls.
```

Comment:
there is comment here that only makes sense based on the prior version of the code that called `.contains(..)`. Can you update the comment accordingly for future code readers?